main/inc/lib/hook/HookAdminBlock.class.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: dbarreto
+ * Date: 19/12/14
+ * Time: 09:45 AM
+ */
+
+class HookAdminBlock extends HookEvent implements HookAdminBlockEventInterface {
+
+    protected function __construct()
+    {
+        parent::__construct('HookAdminBlock');
+    }
+
+    /**
+     * @param int $type
+     * @return int
+     */
+    public function notifyAdminBlock($type)
+    {
+        /** @var \HookAdminBlockObserverInterface $observer */
+        global $blocks;
```

Comment:
Delete global variable use
Update to use setEventData to send variables needed for Hook observer